src/blocks/folder-blocks/homepage/index.tsx

```diff
@@ -8,10 +8,9 @@ export default function (props: FolderBlockProps) {
   const [isLoading, setIsLoading] = useState(false);
 
   const getSiteUrl = async () => {
-    const info = await onRequestGitHubData("repo-info", {
-      owner: props.context.owner,
-      repo: props.context.repo,
-    });
+    const info = await onRequestGitHubData(
+      "https://api.github.com/repos/${props.context.owner}/${props.context.repo}"
```

Comment:
this shouldn't include the domain (`https://api.github.com`), although we should probably handle this use case

Reply:
oh oops I copypasta'd